dfc.providerportal.findanapprenticeship/src/dfc.providerportal.findanapprenticeship.unittests/Functions/GetApprenticeshipsAsProviderTests.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System;
+using System.IO;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Dfc.Providerportal.FindAnApprenticeship.Functions;
+using Dfc.Providerportal.FindAnApprenticeship.Models;
+using Dfc.Providerportal.FindAnApprenticeship.Storage;
+using FluentAssertions;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging.Abstractions;
+using Moq;
+using Xunit;
+
+namespace Dfc.ProviderPortal.FindAnApprenticeship.UnitTests.Functions
+{
+    public class GetApprenticeshipsAsProviderTests
+    {
+        private readonly Mock<IBlobStorageClient> _blobStorageClient;
+        private readonly Mock<Func<DateTimeOffset>> _nowUtc;
```

Comment:
Does this need to be a mock?

Reply:
I prefer it as it makes manipulating the value returned within the tests easier.